Cargo.toml

```diff
@@ -13,15 +13,18 @@ async-trait = "0.1"
 bincode = { version = "1", optional = true }
 ipnet = { version = "2", features = ["serde"] }
 num_enum = "0.7"
-oinq = { git = "https://github.com/petabi/oinq.git", rev = "29ad689d", optional = true }
-quinn = { version = "0.10", optional = true }
+oinq = { git = "https://github.com/petabi/oinq.git", tag = "0.13.0", optional = true }
+quinn = { version = "0.11", features = ["ring"], optional = true }
```

Comment:
It looks like the `ring` feature for the `quinn` crate is being used only in tests, and not in the main library code. If that's the case, it would be better to keep the `ring` feature enabled in the `dev-dependencies` section only.

This way, the `ring` feature and its dependencies won't be compiled into the final binary, keeping the binary size smaller.

Reply:
done